internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -307,6 +365,31 @@ func parseStackNameFromARN(stackARN string) string {
 	return strings.Split(stackARN, "/")[1]
 }
 
+// parseEnvStackFromWorkload retrieves the environment stack's name from the workload's stack name.
+// A workload stack name follows the pattern "{env stack name}-{workload name}".
+func parseEnvStackFromWorkload(stackName string) string {
+	parts := strings.Split(stackName, "-")
+	return strings.Join(parts[:len(parts)-1], "-")
+}
```

Comment:
What if the env stack name is `test-env` and workload name is `my-svc`?

Reply:
oh oh good pointttttt 🤔 let me think about this

Reply:
OK fixed it! I am using the tags in the stack to get the app and env name, let me know your thoughts!